operator/cmd/mesh/manifest-apply.go

```diff
@@ -164,6 +165,9 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 	}
 	_, iops, err := GenerateConfig(inFilenames, setOverlay, force, restConfig, l)
 	if err != nil {
+		if subErr := helm.CheckCompiledInCharts(); subErr != nil {
```

Comment:
Will this fail even if `--charts` is provided?

Reply:
No it won't if --charts $ISTIO/istio/manifests is included.